.github/workflows/pr-tests.yml

```diff
@@ -9,8 +9,13 @@ concurrency:
 
 jobs:
   tests:
-    name: Tests
-    uses: ./.github/workflows/tests.yml
-    with:
-      python-version: "3.9"
-      os: "ubuntu-latest"
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v4
```

Comment:
Is this checkout redundant since `.github/actions/run-unit-tests/action.yml` is also doing a checkout or is the checkout in `.github/actions/run-unit-tests/action.yml` redundant?